subcommands/waves/init.go

```diff
@@ -28,6 +29,14 @@ Use a "fioctl wave complete <wave>" to update all devices (make it globally avai
 Use a "fioctl wave cancel <wave> to cancel a wave (make it no longer available).`,
 		Run:  doInitWave,
 		Args: cobra.ExactArgs(3),
+		Example: `
+To simply initialize a wave run the following:
+$ fioctl wave init -k ~/path/to/keys/targets.only.key.tgz wave-name 4 production
+
+To initialize a wave and prune some old targets at the same time:
+$ fioctl wave init -k ~/path/to/keys/targets.only.key.tgz wave-name 16 production --prune 1,2,3,4
```

Comment:
Maybe, `Start a new wave for the target version 16 and also prune some old production versions:`?